src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
```

Comment:
Why is this its own script? It will only ever be used by the Follower class. I'm not exactly sure how this would even be used in the Follower class as is.